src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -774,3 +773,24 @@ void GraphingCalculator::OnGraphThemeSettingChanged(Object ^ sender, bool isMatc
                                     }
                                 }));
 }
+
+void GraphingCalculator::GraphViewButton_Click(Object ^ sender, RoutedEventArgs ^ e)
+{
+    auto narratorNotifier = ref new NarratorNotifier();
+    String ^ announcementText;
+    if (IsManualAdjustment)
+    {
+        announcementText = AppResourceProvider::GetInstance()->GetResourceString(L"GraphViewManualAdjustmentAnnouncement");
+
+    }
+    else
+    {
+        GraphingControl->ResetGrid();
```

Comment:
Not sure if related to this PR, but it doesn't seem like ResetGrid is always going back to the original perfect auto fit after it was initially moved.